server/api/search.php

```diff
@@ -0,0 +1,19 @@
+<?php
+header('Content-Type: application/json');
+
+include '../env.php';
+require_once "../db.class.php";
+$db = new DB($host, $port, $name, $user, $pass); // From dbinfo.php
+
+switch ($_SERVER['REQUEST_METHOD'])
+{
+   case "GET":
```

Comment:
I was thinking that search.php would be more for the search bar at the top of the application. I think this makes more sense in drivers.php

Reply:
Yeah, sounds good. If you'll need to use the function right away you can move it immediately. If not, I'll move it before the next pull req.

Reply:
Don't worry about it I'm about to do another iteration of hooking up the frontend to the api so I'll take care of it.